<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationConfigure</class>
 <widget class="QDialog" name="CommunicationConfigure">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfiguracja połączenia</string>
  </property>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij okno</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RS232</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plik I/O</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="configurations">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="rs232conf">
    <attribute name="title">
     <string>RS232</string>
    </attribute>
    <widget class="QComboBox" name="rsPort">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="rsBaudRate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="rsDataBits">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="rsParity">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="rsStopBits">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="rsTimeout">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="rsQueryMode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rsConfirm">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zatwierdź</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prędkość:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bity danych:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parzystość: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bity stopu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeoutLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeout (ms):</string>
     </property>
    </widget>
    <widget class="QLabel" name="queryLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tryb zapytania</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fileconf">
    <attribute name="title">
     <string>Plik</string>
    </attribute>
    <widget class="QLabel" name="inputLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Plik wejściowy: </string>
     </property>
    </widget>
    <widget class="QLabel" name="outputLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Plik wyjściowy: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputPath">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="outputPath">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="inputFileChoose">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="outputFileChoose">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmFile">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zatwierdź</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij port</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>testButton</sender>
   <signal>clicked()</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>testConnection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>configurations</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rsConfirm</sender>
   <signal>clicked()</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>confirmRS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputFileChoose</sender>
   <signal>clicked()</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>getInputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputFileChoose</sender>
   <signal>clicked()</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>getOutputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmFile</sender>
   <signal>clicked()</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>confirmFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rsQueryMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>checkTimeoutEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CommunicationConfigure</receiver>
   <slot>closeActivePort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>testConnection()</slot>
  <slot>confirmRS()</slot>
  <slot>getInputFile()</slot>
  <slot>getOutputFile()</slot>
  <slot>confirmFile()</slot>
  <slot>checkTimeoutEnabled()</slot>
  <slot>closeActivePort()</slot>
 </slots>
</ui>
